### DISCUSSION TITLE
Submit button option for default UI's search input

### DISCUSSION THREAD

**Discussion by olets** [Pagefind/pagefind#480] [Ideas]
I'd like the option of disabling the live search update, and instead waiting for user interaction with a submit button. Is that something you'd consider for the default UI?

**Comment by bglw** [comment]
It isn't something I'm likely to work on for the default UI myself, though I would be open to pull requests for it.

The (upcoming) [Modular UI](https://www.npmjs.com/package/@pagefind/modular-ui) would be an even better path at the moment, as you could use mostly off the shelf modules with a custom search module.
